modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf/pom.mustache

```diff
@@ -154,12 +154,20 @@
         <version>${jackson-jaxrs-version}</version>
         <scope>compile</scope>
     </dependency>
+{{#java8}}
     <dependency>
-       <groupId>com.fasterxml.jackson.datatype</groupId>
-       <artifactId>jackson-datatype-joda</artifactId>
-       <version>${jackson-jaxrs-version}</version>
-       <scope>compile</scope>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-jsr310</artifactId>
+        <version>${jackson-jaxrs-version}</version>
+    </dependency>
+{{/java8}}
+{{^java8}}
```

Comment:
There is joda ( {{#joda}} ) tag for joda

Reply:
Does it make sense to use it? What happens if both are enabled?

Look for `{{^java8}}` and you'll find several other files, where this is already in place, e.g.:
https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/JavaJaxRS/jacksonJsonProvider.mustache